packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -680,6 +680,13 @@ export const formatters: Formatters = {
       return `Certificate of type [${cert_type}] issued for [${user}]`;
     },
   },
+  [eventCodes.SESSION_RECORDING_VIEW]: {
+    type: 'session.recording.view',
+    desc: 'Session Recording Viewed',
+    format: ({ sid, user }) => {
+      return `User [${user}] viewed a session recording [${sid}]`;
```

Comment:
I feel like "accessed" a session recording is a better term than "viewed".

(You can download sessions via tsh, for example)

Reply:
Ah yes I was going to rename the event as per your other PR review comment - I'll change the wording too